### PR TITLE
Update Chinese trsl

### PR DESCRIPTION
**Pull request by ChuijkYahus**

### CODE REVIEW

**Comment by ChuijkYahus** [comment]
patchouli book also gets translated!
帕秋莉手册也翻了！

**Comment by Lgmrszd** [comment]
Thank you, will add in next update!

**Comment by ChuijkYahus** [comment]
Thank you so much!

Also if there are new translation keys in the next update, I'm going to update them in this pr, or a new pr if this one is merged.

File: src/main/resources/assets/compressedcreativity/lang/zh_cn.json [hunk 1]
```diff
@@ -26,31 +45,30 @@
...
+  "compressedcreativity.tooltip.air_production": "产出空气：",
+  "compressedcreativity.tooltip.air_usage": "消耗空气：",
+  "compressedcreativity.tooltip.compressed_air_engine.idle_1": "这台机器目前处于空闲状态：",
   "compressedcreativity.tooltip.compressed_air_engine.idle_2": "这台机器即使不工作也会消耗空气",
-  "compressedcreativity.tooltip.not_enough_pressure": "压力需求:",
+  "compressedcreativity.tooltip.not_enough_pressure": "压力需求：",
   "compressedcreativity.tooltip.not_enough_pressure_2": "这台机器需要更大的压力才能工作",
-  "compressedcreativity.tooltip.pressure": "压力:",
+  "compressedcreativity.tooltip.pressure": "压力：",
   "compressedcreativity.tooltip.pressure_summary": "压力统计:",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/compressedcreativity/lang/zh_cn.json]:
```suggestion
  "compressedcreativity.tooltip.pressure_summary": "压力状态：",
```

**Comment by Lgmrszd** [comment]
Hello, could you update this PR?

**Comment by ChuijkYahus** [comment]
back to update now!

**Comment by ChuijkYahus** [comment]
Sorry for the wait!

The manual ingame doesn't yet have an implemented Chinese translation, so I think terminology checks from the person working on that may be needed.
ping @dovisutu

File: src/main/resources/assets/pneumaticcraft/patchouli_books/book/zh_cn/entries/copper_backtank.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    {
+      "type": "text",
+      "text": "方块形态和物品形态的$(item)铜背罐$()都有了全新功能！$(br)其最多可充至3 bar。$(br2)在方块形态下，其底部会与各种供气端口连接，可用作简易空气压缩机。多余的空气会被直接排出。"
+    },
+    {
+      "type": "image",
+      "images": ["compressedcreativity:textures/gui/bt_block.png"],
+      "border": true,
+      "title": "铜背罐",
+      "text": "铜背罐用作压缩器"
```
Comment (dovisutu) [inline src/main/resources/assets/pneumaticcraft/patchouli_books/book/zh_cn/entries/copper_backtank.json]: compressor -> 压缩机

File: src/main/resources/assets/pneumaticcraft/patchouli_books/book/zh_cn/entries/copper_backtank.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    {
+      "type": "text",
+      "text": "在物品形态下，其余$(l:pneumaticcraft:components/air_canister)空气罐$()功能类似。也即为其充气，就和任何其他能装压缩空气的物品一样。"
```
Comment (dovisutu) [inline src/main/resources/assets/pneumaticcraft/patchouli_books/book/zh_cn/entries/copper_backtank.json]: typo 🤣 also minor changes
```suggestion
      "text": "在物品形态下，其与$(l:pneumaticcraft:components/air_canister)空气罐$()功能类似。也即可以充气，就和任何压缩空气容器一样。"
```

File: src/main/resources/assets/pneumaticcraft/patchouli_books/book/zh_cn/entries/intro.json [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    },
+    {
+      "type": "spotlight",
+      "item": "compressedcreativity:rotational_compressor",
+      "text": "$(l:pneumaticcraft:rotational_compressor)旋转压缩机$()是一种使用机械动力的旋转力产生压力的压缩机。"
+    },
+    {
+      "type": "spotlight",
+      "item": "create:copper_backtank",
+      "text": "The $(l:pneumaticcraft:copper_backtank)Copper Backtank$() received additional functionality, having pneumatic capabilities in both Item and Block form."
```
Comment (dovisutu) [inline src/main/resources/assets/pneumaticcraft/patchouli_books/book/zh_cn/entries/intro.json]: Seems to have missed several terms in this file though...

**Review by dovisutu — COMMENTED** [review]
Generally speaking, the terminology is almost fine, except several points below.
I don't think I'm knowledgable in Create, so maybe some mistakes there.

btw I would suppose I'm working on the translation for PnC (and its manual), no certain ETA though.

File: src/main/resources/assets/pneumaticcraft/patchouli_books/book/zh_cn/entries/intro.json (hunk 4 above)
Comment (ChuijkYahus) [reply src/main/resources/assets/pneumaticcraft/patchouli_books/book/zh_cn/entries/intro.json]: Strange, seems this file didn't get updated correctly...
Gonna try to fix it.

Comment (ChuijkYahus) [reply src/main/resources/assets/pneumaticcraft/patchouli_books/book/zh_cn/entries/intro.json]: I guess it's OK now...

**Comment by ChuijkYahus** [comment]
Issues fixed (at least the mentioned ones) and wording changes for readability.

**Comment by Lgmrszd** [comment]
Is this okay to merge?

**Comment by ChuijkYahus** [comment]
> Is this okay to merge?

Skimmed through once more.
I think it's OK to merge now.

**Comment by dovisutu** [comment]
Yeah, the text now also looks generally fine, at least when I eyeball it. Should be able to merge now......

**Pull request merged**